### DISCUSSION TITLE
Turbo Extension for Other Editors

### DISCUSSION THREAD

**Discussion by versecafe** [vercel/turborepo#8985] [Ideas]
### Goals

Right now turbo only has a vscode extension but given the simplicity of it as an LSP it would be nice to bring the extension to more editors such as Zed & jetbrains IDEs

1. Zed Extension
2. Jetbrains Extension
3. NeoVim Extension

### Non-goals

1. Expanding the features of the current turbo extension
2. Modifying the extension in any given way

### Background

The current `turbo scan` command throws explicit warnings about the LSP not running when using any editor besides vscode, and adding to other editors just improves the all around experience.

### Proposal

A simple LSP based language extension for the listed editors or any others with an extension system supporting language servers.

**Comment by anthonyshew** [comment]
I don't think Zed has extensions today - but happy to get a PR reviewed for the others!
